docs/current_docs/quickstart/snippets/daggerize/php/src/HelloDagger.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Attribute\Doc;
+use Dagger\Container;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class HelloDagger
+{
+    #[DaggerFunction]
+    #[Doc('Publish the application container after building and testing it on-the-fly')]
+    public function publish(Directory $source): string
+    {
+        $this->test($source);
+
+        return $this
+            ->build($source)
+            ->publish('ttl.sh/myapp-'.rand(0, 10000000));
```

Comment:
While comparing I noticed they all use `ttl.sh/myapp` **except Go:**

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/daggerize/go/main.go#L21

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/daggerize/python/__init__.py#L14

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/daggerize/typescript/index.ts#L12

Should this be changed?
This could entirely be done in a different PR, but I thought it was worth pointing out.

Reply:
Great catch, thanks! I've opened https://github.com/dagger/dagger/pull/9521 to fix this in main, and then will rebase and fix the PHP snippets in this PR.

Reply:
Fixed now